apps/web/locales/en.json

```
@@ -40,6 +40,24 @@
 		"SCREENSHOTS": "Screenshots",
 		"TIME_LOG": "Time log",
 		"TO_DO": "To-do",
+		"PRODUCTIVE": "Productive",
+		"NEUTRAL": "Neutral",
+		"UNPRODUCTIVE": "Unproductive",
+		"ACTIVITIES": "activities",
+		"MEMBERS": "members",
```

Biome (1.9.4)

[error] The key MEMBERS was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)